app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -111,9 +111,33 @@ controlPlane:
   injectorFailurePolicy: Fail
 
   service:
+    admissionServer:
+      # -- Port on which Https admission server Service is exposed on Node for service of type NodePort
+      nodePort: 30443
+
+    apiServer:
```

Comment:
Can we add just `apiServer` for now, since it's likely the only one being accessed from outside the cluster?

Reply:
Done

Reply:
Sorry, I meant for this service. The global sync nodeport can stay 😅